index.html

```diff
@@ -2329,7 +2329,68 @@ <h2 id="OfflineAudioCompletionEvent">
           value explicitly. <em>block-size</em> is defined to be 128
           sample-frames which corresponds to roughly 3ms at a sample-rate of
           44.1KHz.
-        </p>
+        </p><a>AudioNode</a>s can be created in two ways:
+        <ul>
+          <li>First, an <a>AudioNode</a> can be created by calling the
+          corresponding <dfn>factory method</dfn> on the <a>AudioContext</a>,
+          passing in arguments. The <a>AudioNode</a> is associated with the <a>
+            AudioContext</a> it was created from.
+          </li>
+          <li>Second, an <a>AudioNode</a> can be create by calling its
+          constructor, passing in the <a>AudioContext</a> with which the
+          <a>AudioNode</a> being constructed will be associated with. A second
+          argument is a dictionnary that can contain attributes specific to the
```

Comment:
s/dictionnary/dictionary/

Should we also say that the second argument is (sometimes) optional?